frontend/src/Movie/Index/Posters/MovieIndexPoster.tsx

```diff
@@ -214,10 +233,33 @@ function MovieIndexPoster(props: MovieIndexPosterProps) {
         <div className={styles.title}>{qualityProfile.name}</div>
       ) : null}
 
+      {showCinemaRelease && inCinemas ? (
+        <div className={styles.title}>
+          <Icon name={icons.IN_CINEMAS} />{' '}
+          {getRelativeDate(inCinemas, shortDateFormat, showRelativeDates, {
+            timeFormat,
+            timeForToday: false,
+          })}
+        </div>
+      ) : null}
+
+      {showReleaseDate && releaseDate && releaseDateType ? (
```

Comment:
I believe this would work as you would never get `releaseDate` without `releaseDateType` 

```suggestion
      {showReleaseDate && releaseDate ? (
```